### PR TITLE
Bulkscan caseworker for CCD

### PR DESCRIPTION
**Pull request by luigibk**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPE-708

### Change description ###
Create jurisdiction in CCD to run tests for bulk scanning

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,14 +51,28 @@ To stop the environment use the same script, just make sure to pass the `local`
...
+Once the containers are up, we can then create a caseworker user to login into CCD. 
 
-By default `caseworker-sscs` role is assigned to the user.
-If you want to change the role pass in appropriate role while executing the script. User role should exists in Idam.
+The required user should have the `caseworker-bulkscan` role. This matches the CCD definition loaded at the next stage.
 
-  ```bash
-   $ ./bin/create-case-worker.sh
-  ```
+Before running the `docker/ccd-definition-import/scripts/caseworker-bulkscan-idam.sql` sql script you need to edit it and
```
Comment (timja) [inline README.md]: This should be automated with a container that adds the user, 
Password should not be different for local environment as it adds unneeded complexity

File: docker/ccd-definition-import/scripts/upload-definition.sh [hunk 2]
```diff
@@ -24,7 +24,7 @@ export BULK_SCAN_ORCHESTRATOR_BASE_URL=http://bulk-scan-orchestrator:5000 # Serv
 userToken=$(sh ./scripts/idam-authenticate.sh ${IMPORTER_USERNAME} ${IMPORTER_PASSWORD} ${IDAM_URI} ${REDIRECT_URI} ${CLIENT_SECRET})
 
 # add ccd role
-/scripts/add-ccd-role.sh "caseworker-sscs" "PUBLIC" "${userToken}"
+/scripts/add-ccd-role.sh "caseworker-bulkscan" "PUBLIC" "${userToken}"
 /scripts/add-ccd-role.sh "caseworker-sscs-systemupdate" "PUBLIC" "${userToken}"
```
Comment (timja) [inline docker/ccd-definition-import/scripts/upload-definition.sh]: these can be removed

Comment (luigibk) [reply docker/ccd-definition-import/scripts/upload-definition.sh]: Are you sure? I don't think it works without this.

File: README.md (hunk 1 above)
Comment (luigibk) [reply README.md]: I discussed this with @nitinprabhu and we got to the conclusion that better safe than sorry and so the password is available in vault.

File: docker/ccd-definition-import/scripts/upload-definition.sh (hunk 2 above)
Comment (timja) [reply docker/ccd-definition-import/scripts/upload-definition.sh]: we're changing the jurisdiction from sscs to bulk scan so yes this is the time to remove it and fix any issues

File: README.md (hunk 1 above)
Comment (timja) [reply README.md]: this is silly complexity and absolutely not needed for local development

Comment (nitinprabhu) [reply README.md]: For local it should be fine as Tim suggested but for AAT and DEMO password needs to be in vault ?

Comment (nitinprabhu) [reply README.md]: To run functional tests we need sql script to be executed on aat and demo

Comment (luigibk) [reply README.md]: Yes, see readme file, both of you :-)

File: docker/ccd-definition-import/scripts/caseworker-bulkscan-idam.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
+INSERT INTO role (name, display_name) VALUES ('caseworker-bulkscan', 'Case Worker Bulk Scan');
```
Comment (KevinJCross) [inline docker/ccd-definition-import/scripts/caseworker-bulkscan-idam.sql]: cant we make this part of the startup flyway scripts for our version of ccd ?

**Pull request merged**